src/directives/events.js

```diff
@@ -1,48 +1,48 @@
-import animations from './animations';
+import animations from "./animations";
```

Comment:
Please use single quote `'` over `"` please see semistandard rules. Apply to all.